freeipa/src/main/java/com/sequenceiq/freeipa/controller/FreeIpaV1Controller.java

```diff
@@ -308,4 +309,10 @@ public FlowIdentifier retry(@ResourceCrn @NotEmpty String environmentCrn) {
         String accountId = crnService.getCurrentAccountId();
         return retryService.getRetryableFlows(environmentCrn, accountId);
     }
+
+    @Override
+    @CheckPermissionByResourceCrn(action = EDIT_ENVIRONMENT)
+    public void changeImageCatalog(@ResourceCrn @NotEmpty String environmentCrn, @Valid @NotNull ChangeImageCatalogRequest changeImageCatalogRequest) {
+        // FIXME: Implement this!
```

Comment:
API first approach so we can work in parallel on the CLI.
It is going to be implemented as a part of CB-14310